lessons/05-html-linking/documents/resume.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+
+<head>
+    <title> resume</title>
+
+<body>
+    <h1>Resume</h1>
+    <ul>
+        <li><a href="../index.html"> Home</a></li>
+        <li><a href="../news.html"> News</a></li>
+        <li><a href="../about.html">About</a></li>
+        <li><a href="../contact.html">Contact</a></li>
+    </ul>
+    <h1>Personal information</h1>
+    <hr>
+    <p>
+        <b>Name:</b>Azadeh <br>
+        <b>Adress:</b> my home <br>
+        <b>Phone:</b>45678 <br>
+    </p>
+    <h1>Experience</h1>
+    <hr>
+    <b>Top Dog</b>- <em>May17-Present</em>
+    <p> Lorem ipsum dolor sit amet consectetur adipisicing elit. In reiciendis amet earum quae repudiandae temporibus
+        suscipit optio ullam iste. Odit praesentium totam iure eveniet ab impedit voluptatem odio placeat quidem!</p>
+</body>
+</head>
```

Comment:
Oops! We're just missing a few things

1. The outer `<html>` tags surrounding everything
2. The `<head>` tag here also has the `<body>` inside of it. We want to close the head tag after the `<title>`